src/shared/messaging/port-rpc.js

```diff
@@ -275,7 +290,7 @@ export class PortRPC {
 
     const seq = this._sequence++;
     const finalArg = args[args.length - 1];
-    if (typeof finalArg === 'function') {
+    if (isCallback(finalArg)) {
```

Comment:
This change alters the inferred type of `finalArg` slightly. Without it, TS complains about the `set(...)` call below.